app/views/layouts/_navbar.html.erb

```diff
@@ -1,5 +1,32 @@
-<ul class="navbar">
-  <li><a href="/">Home</a></li>
-  <li><a href="/users">Users</a></li>
-  <li><a href="/posts">Posts</a></li>
-</ul>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Navbar</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="/">PeopleConnector</a>
+    </div>
+    <ul class="nav navbar-nav">
+      <li class="active"><a href="/">Home</a></li>
+      <li><a href="/users">Users</a></li>
+      <li><a href="/posts">Posts</a></li>
+    </ul>
+    <ul class="nav navbar-nav navbar-right">
+      <li><a href="/"><span class="glyphicon glyphicon-user"></span> New Post</a></li>
```

Comment:
`New Post` is still redirecting to new user page. Maybe change the `glyphicon-user` icon too and see if there is an icon for post.
I suggest having both `New Post` and `New User` here since you are removing the links from the tables.

Reply:
```  <div class="container-fluid">
    <div class="navbar-header">
      <a class="navbar-brand" href="/">PeopleConnector</a>
    </div>
    <ul class="nav navbar-nav">
      <li><a href="/users">Users</a></li>
      <li><a href="/posts">Posts</a></li>
    </ul>
    <ul class="nav navbar-nav navbar-right">
      <li><a href="/users/new"><span class="glyphicon glyphicon-user"></span> New User </a></li>
      <li><a href="/posts/new"><span class="glyphicon glyphicon-edit"></span> New Post </a></li>
      <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login </a></li>
    </ul>
  </div>
```